docs/content/adoption/companies_experience.md

```diff
@@ -33,3 +33,19 @@ for screens which implemented by using that components.
 
 
 
+## Kaspersky
+
+`UI testing`
+
+
+: `Write:` [Kaspresso](https://github.com/KasperskyLab/Kaspresso) <br>
+`Who:` QA and developers<br>
```

Comment:
right now I start thinking that we should change template a bit for `Who writes the tests` may be. Just because right now it's not so clear what does it mean :D

Reply:
yep, let's change it

Reply:
fixed